src/guide/2-model-definition/2-environment.rst

```diff
@@ -67,6 +67,27 @@ Any arithmetic or enum type can be used as an environment property
     env.newPropertyChar("c_prop", 'g', True)  # Create constant char property 'c_prop', with value 'g'
 
 
+Macro Environmental Properties
+---------------------------------
+Macroscopic environment properties are less efficient than the regular environment properties described above, as they reside in slower memory. However, they are useful when a model requires a large amount of environment data to be stored, or to perform simple tasks between agents, such as counting and exchanging swapping data, without needing to send and recieve messages across multiple simulation layers.
```

Comment:
Might be better to invert this sentence, start with what they are used for rather than what they are bad for.

Reply:
Makes, sense, will do. Thanks